### PR TITLE
[master < T1128] Implement RemoveProperty for node cpp api

### PR DESCRIPTION
**Pull request by mpintaric55334**
A need for RemoveProperty function was found when implementing custom C++ Query module. So it was implemented.
[master < Task] PR
- [ ] Check, and update documentation if necessary
- [ x] Provide the full content or a guide for the final git message

To keep docs changelog up to date, one more thing to do:
- [ x] Write a release note here, including added/changed clauses
Users can now use RemoveProperty() function for class Node when writing custom query modules in C++ API.
- [ x] Tag someone from docs team in the comments

### CODE REVIEW

**Comment by mpintaric55334** [comment]
@vpavicic will start writing the docs for this when we know where to write for C++ API

File: include/mgp.hpp [hunk 1]
```diff
@@ -636,6 +636,9 @@ class Node {
   /// @brief Sets the chosen property to the given value.
   void SetProperty(std::string property, Value value);
 
+  /// @brief Sets the chosen property to the given value.
```
Comment (antoniofilipovic) [inline include/mgp.hpp:639]: Removes the property, I think this is left from copy paste :smile:

File: tests/unit/cpp_api.cpp [hunk 2]
```diff
@@ -560,3 +560,14 @@ TYPED_TEST(CppApiTestFixture, TestMapErase) {
...
   ASSERT_EQ(map.Size(), 0);
 }
+TYPED_TEST(CppApiTestFixture, TestNodeRemoveProperty) {
+  mgp_graph raw_graph = this->CreateGraph(memgraph::storage::View::NEW);
+  auto graph = mgp::Graph(&raw_graph);
+  auto node = graph.CreateNode();
+
+  int64_t int1 = 100;
+  mgp::Value value = mgp::Value(int1);
+  node.SetProperty("key", value);
```
Comment (antoniofilipovic) [inline tests/unit/cpp_api.cpp:570]: Can you add one assert in the middle between setting and removing that we have actually set the property?

Comment (antoniofilipovic) [reply tests/unit/cpp_api.cpp:570]: So, size==1, or something like that

File: tests/unit/cpp_api.cpp [hunk 3]
```diff
@@ -560,3 +560,14 @@ TYPED_TEST(CppApiTestFixture, TestMapErase) {
   map.Erase("2");
   ASSERT_EQ(map.Size(), 0);
 }
+TYPED_TEST(CppApiTestFixture, TestNodeRemoveProperty) {
+  mgp_graph raw_graph = this->CreateGraph(memgraph::storage::View::NEW);
+  auto graph = mgp::Graph(&raw_graph);
+  auto node = graph.CreateNode();
+
+  int64_t int1 = 100;
+  mgp::Value value = mgp::Value(int1);
```
Comment (antoniofilipovic) [inline tests/unit/cpp_api.cpp:569]:
```suggestion
  mgp::Value value{int1};
```

**Review by antoniofilipovic — CHANGES_REQUESTED** [review]
Few changes

**Review by antoniofilipovic — APPROVED** [review]

**Pull request merged**